decidim-core/app/cells/decidim/content_blocks/highlighted_content_banner/show.erb

```diff
@@ -7,7 +7,7 @@
           <%= translated_attribute current_organization.highlighted_content_banner_title %>
         </h1>
         <span class="text-highlight">
-          <%= decidim_sanitize_editor translated_attribute current_organization.highlighted_content_banner_short_description %>
+          <%= decidim_sanitize translated_attribute current_organization.highlighted_content_banner_short_description %>
```

Comment:
@andreslucena This decidim sanitize is actually breaking the validation. Either remove the `_editor` part, either convert the span to div. 

Close this after you have read.

Reply:
I've messed up the answer - lets convert the span to div and leave the _editor please!